lib/generators/react_on_rails/templates/client/index.jade

```diff
@@ -0,0 +1,14 @@
+doctype html
```

Comment:
For all the generated files, let's leave a comment of the URL:

https://github.com/shakacode/react-webpack-rails-tutorial/blob/master/client/index.jade

The reasons are:

1. To see where to get more documentation
2. Inevitably, the gem will get out of sync with the tutorial